cli/jobs/spark-standalone-jobs/spark_job_conda_config.yml

```diff
@@ -4,7 +4,7 @@ type: spark
 code: ./src_files
 description: "spark job example"
 environment:
-  conda_file: ./conda.config
+  conda_file: ./conda.yml
 compute: azureml:synapsecompute
```

Comment:
did we create the compute in the CI workspace?

Reply:
yep
https://ml.azure.com/compute/synapsecompute/details?wsid=/subscriptions/6560575d-fa06-4e7d-95fb-f962e74efd7a/resourceGroups/azureml-examples/providers/Microsoft.MachineLearningServices/workspaces/main&flight=NotebookSpark&tid=72f988bf-86f1-41af-91ab-2d7cd011db47